File: federaproApi/src/main/java/com/example/federaproapi/karting/dto/KartingEstadisticaPilotoDTO.java
package com.example.federaproapi.karting.dto;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
public class KartingEstadisticaPilotoDTO {
    private String nombrePiloto;
    private Integer idCarrera;
    private Integer posicion;
    private BigDecimal tiempoTotal;
    private Integer vueltas;
}

File: FederaProDesktop/Karting/DTOs/KartingEstadisticaPilotoDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FederaProDesktop.Karting.DTOs
{
    public class KartingEstadisticaPilotoDTO
    {
        public string NombrePiloto { get; set; }
        public int IdCarrera { get; set; }
        public int Posicion { get; set; }
        public decimal TiempoTotal { get; set; }
        public int Vueltas { get; set; }
    }
}
